kafka-connect-s3/src/main/java/io/confluent/connect/s3/extensions/UdxStreamPartitioner.java

```diff
@@ -175,16 +174,16 @@ public String encodePartition(SinkRecord sinkRecord) {
     log.info("encoding partition with UdxStreamPartitioner...");
     log.info("Parsing value...");
 
-    String payload = sinkRecord.value().toString();
+    String stringPayload = sinkRecord.value().toString();
     ObjectMapper mapper = new ObjectMapper();
-    AvroPayload avroPayload;
+    JsonPayload jsonPayload;
     try {
-      avroPayload = mapper.readValue(payload,AvroPayload.class);
+      jsonPayload = mapper.readValue(stringPayload, JsonPayload.class);
     } catch (JsonProcessingException e) {
-      return generateCorruptAvroPayloadPartition();
+      return generateCorruptJsonPayloadPartition();
     }
 
-    String jsonStringValue = avroPayload.getPayload();
+    String jsonStringValue = jsonPayload.getPayload();
     log.info("Value: " + jsonStringValue);
```

Comment:
I wonder if it might be useful to add some context to each of these logs, e.g. adding the partition number / offset and (when we have it later on in this code block) the stream uuid and entity id? If we ever have a problem where we need to trawl through the logs, having that extra context might be troubleshooting a whole lot easier